### PR TITLE
i18n/ja Add Japanese translation

### PR DESCRIPTION
**Pull request by inductor**
Hello! I'd like to push some of changes since I saw the "big" issue on https://github.com/parcel-bundler/website/issues/94 and I really would like to help the project :)

### CODE REVIEW

File: src/i18n/ja/index.html [hunk 1]
```diff
@@ -0,0 +1,236 @@
...
+  <span class="hljs-comment">/* Reference an image file */</span>
+  <span class="hljs-attribute">background</span>: <span class="hljs-built_in">url</span>(<span class="hljs-string">'./images/background.png'</span>);
+  <span class="hljs-attribute">color</span>: red;
+}</code></pre>
+          </div>
+        </div>
+
+        <p>
+          <code>parcel index.html</code>
+          とタイプするだけで、開発要サーバーが立ち上がります。JavaScript、CSS、画像などの読み込みもこれだけで動きます！
```
Comment (AkihiroSuda) [inline src/i18n/ja/index.html]: Misconversion: 開発要

**Comment by inductor** [comment]
@zacky1972 @Pittan Would you review? :)

I'm planning to remove the WIP since I done with the basic translation on the main page(index.html). (refer to: https://github.com/parcel-bundler/website/issues/376)

File: src/i18n/ja/index.html [hunk 2]
```diff
@@ -60,57 +62,60 @@ <h1><img class="logo" src="assets/logo.svg" alt="Parcel" /></h1>
...
-            even after a restart.
+            Parcel
+            はマルチコアコンパイルを有効にするためにワーカープロセスを使い、再起動後でも高速に動くようにファイルシステムのキャッシュを持ちます。
           </p>
         </section>
         <section>
-          <h3>📦 Bundle all your assets</h3>
-          <p>Parcel has out of the box support for JS, CSS, HTML, file assets, and more - no plugins needed.</p>
+          <h3>📦 全アセットをまとめます</h3>
+          <p>Parcel 形にとらわれず、 JS、CSS、HTML、ファイルアセットなどをプラグインなしでサポートします。</p>
```
[the remark this answers is not in the document]
Comment (inductor) [reply src/i18n/ja/index.html]: 是非このinsert suggestion機能も使ってみてください 😄

**Comment by Pittan** [comment]
@devongovett 
This pull request looks good to me, but I don't have a write access.
Could you approve this pull request?

**Pull request merged**